.gitignore

```diff
@@ -45,4 +45,7 @@ gradle-app.setting
 .project
 
 # JDT-specific (Eclipse Java Development Tools)
-.classpath
\ No newline at end of file
+.classpath
+
+data.txt
```

Comment:
What is in data.txt?

Reply:
It's a generated file that `test/resources/gen-producer-data.sh` outputs to pass thousands of entries, etc

Reply:
I like to keep all generated things in `build` so it is easy to clean. The only downside is, if generating it takes a lot of time and the generation hardly ever changes, then it is kind of annoying that a clean will throw it away.

Reply:
`data.txt` is an artifact generated manually to facilitate a manual test of a lot of data being passed via gRPC.  I've removed those scripts and the reference here.